spec/person_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './../person'
+require_relative './../decorators/nameable'
+
+describe 'Test person object' do
+  person = Person.new(18)
+
+  it 'takes parameters and returns a Nameable object' do
+    expect(person).to be_kind_of(Nameable)
+  end
+
+  it 'takes parameters and returns a Person object' do
+    expect(person).to be_instance_of(Person)
+  end
+
+  it "person name should be 'Unknown' when not defined" do
+    expect(person.name).to eql 'Unknown'
+  end
```

Comment:
- [ ] It would be nice if you could add more tests for the person_spec.rb file as the actual person.rb file class has methods for which you are expected to write tests: `can_use_services`, `orrect_name`, `add_rental`, and `of_age` methods. Kindly write tests for these methods as that is what the requirements asked, to write tests for all methods in the corresponding classes 👍🏽